### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Akhtar-Joomun**
Media queries used for a desktop version website.
Layout changes from mobile version to desktop version at the width of 992px

### CODE REVIEW

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Ak0706, 

Good job so far!
Excellente project!!! But we need to make some changes to improve it.

mario_ladders

## Good points 👍 
- No Linters error ✔️ 
- Your mobile design is excellent ✔️ 

## Aspects to improve ♻️

- [ ] The mobile design is good, but according to this project, you need to have desktop size too. I recommend you to check the Figma template to know how should look on the desktop because, at this point, both are the same.
- This is the problem in tablet (with blank spaces on both sides), when you see it, is the same as mobile but, it should be different.
image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ak0706** [comment]
Ok, thank you for the review, but I still have some questions.

I made the mistake of removing the width from the mobile version.
But for the desktop version, it was set to 100%. Isn't that better than setting it to 1440px?
Otherwise, it doesn't fit my screen. 

Please let me know what I should do.
Thanks again.

File: index.html [hunk 1]
```diff
@@ -29,6 +29,17 @@
...
+                            </svg></a></li>
+                </ul>
+            </div>
```
Comment (Shaher-11) [inline index.html]: Kindly align the nav elements and the rest of the section as the given design

### Original design 
right1

#### Your project
wrong1

File: index.html [hunk 2]
```diff
@@ -243,28 +259,28 @@ <h2 class="about-me-title">About me</h2>
...
                 </ul>
             </li>
-            <li class="down">
+            <li class="down skills">
                 <h2 class="list-title">Skills</h2>
-                <ul class="lists">
+                <ul class="lists codekit">
                     <li>Codekit</li>
                     <li>Github</li>
                     <li>Codepen</li>
```
Comment (Shaher-11) [inline index.html]: ### Original design
[Orignal design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)
#### Your Project
wrong2


wrong5


wrong4

wrong3

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @Ak0706 

### Great work on making the changes requested by the previous reviewer:clap::clap::clap:
You've done a good job implementing some of the requested changes but there are still some which aren't done yet,  so we need to implement those changes to get the project approved:muscle::muscle::muscle:

 ## Required Changes ♻️

- [x] You need to make sure that your desktop version looks like the one in the Figma [design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).
- [x] Check the comments below:point_down:


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi @Ak0706, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**